code/game/objects/effects/overlays.dm

```diff
@@ -135,6 +135,7 @@
 /obj/effect/overlay/temp/decoy
 	desc = "It's a decoy!"
 	duration = 15
+	mouse_opacity = 0
```

Comment:
These aren't really decoys, the path is really just a lie; they're trail effects that fade out really quickly and it wouldn't be fun to accidentally click on one.